mev-rs/src/blinded_block_relayer/api/client.rs

```diff
@@ -27,6 +27,7 @@ impl BlindedBlockRelayer for Client {
     async fn submit_bid(
         &self,
         signed_submission: &SignedBidSubmission,
+        cancellation_enabled: bool,
     ) -> Result<SignedBidReceipt, Error> {
         let response = self.api.http_post("/relay/v1/builder/blocks", signed_submission).await?;
```

Comment:
want to round this out on the client?

I really like that you added this to the server part

we can optionally add a query param according to the spec: https://flashbots.github.io/relay-specs/#/Builder/submitBlock

here's an example to add the param: https://github.com/ralexstokes/beacon-api-client/blob/d838d930f80fdfcadfe32147bcb2e805aec074bc/src/api_client.rs#L233

Reply:
Yeah! Sounds like a great time.

Reply:
@ralexstokes one point of clarification. This involves replacing the `BeaconApiClient` with a reqwest::Client correct?  The inner client to in BeaconApiClient is not public.

Reply:
@ralexstokes BeaconApiClietn doesn't publicly expose the underlying reqwest::Client. We'd need to upstream or create a new client struct to add an additional query parameter. Want to make sure I have the right idea before I continue.

Reply:
yeah good point...


do you want to make a PR to `beacon-api-client` to just make `endpoint` and `client` `pub`?

then you can update the dep here and just do it directly here, rather than use the exported functions on `Client`

I took a look but think this probably best rather than trying to make another POST method that also takes query params

Reply:
Sounds good opened a pr https://github.com/ralexstokes/beacon-api-client/pull/77